### ISSUE TITLE
Miner registration blocks do not appear to be applying

### ISSUE THREAD

**Issue by obskein** [xgt-network/xgt#7]
Operations currently emitted from the XGT-wallet command

 ```ruby
register_miner("address", "recovery_public", "recovery_private")
```

Are not currently being applied correctly to the chain, precluding new miners.

**Comment by obskein** [comment]
This is likely caused by micro-fork TX loss due to #5 - @obskein to validate this issue goes away after resolution of #5.

**Comment by obskein** [comment]
This issue was resolved in 1.1.0; there was a check that blocked correct syncing between nodes.
